src/common/options/mds-client.yaml.in

```diff
@@ -245,6 +245,14 @@ options:
   fmt_desc: If set to ``true``, clients read data directly from OSDs instead
     of using a local page cache.
   with_legacy: true
+- name: client_debug_force_send_request_to_rank0
+  type: bool
+  level: dev
+  default: false
+  services:
+  - mds_client
+  fmt_desc: If set to ``true``, clients will always send client requests to rank 0.
```

Comment:
I've long wanted a config like this in the past but I would prefer if it could be set to any rank (i.e. make it an `int`).